packages/expo-doctor/src/checks/ProjectSetupCheck.ts

```diff
@@ -29,12 +32,12 @@ export class ProjectSetupCheck implements DoctorCheck {
         )
       ) {
         issues.push(
-          'This project contains local Expo modules, but the android/ios folders inside the modules are gitignored. These files are required to build your native module into your app. Use patterns like "/android" and "/ios" in your .gitignore file to exclude only the top-level android and ios folders.'
+          `The "android" and/or "ios" directories (./modules/your-module/[android|ios]) for local Expo modules are gitignored, and they should not be. This is often due to overly general gitignore rules. Use patterns like "/android" and "/ios" in your .gitignore file to exclude only the top-level "android" and "ios" directories, and not those in the modules directory.`
```

Comment:
say directory instead of folder. directory is a filesystem concept. folder is a UI metaphor